feat(docs): Update docs and images for auto-assignment (#9291)
getsentry/sentry-docs: docs/product/issues/suspect-commits/index.mdx

In [sentry.io](https://sentry.io):

1. Go to **Settings > Integrations**.
2. Click the integration you’d like to use.
3. Configure the integration if you haven’t already.

> 4. Add the repository you'd like to use.

Reviewer: Not a big deal, but I would err on the side of not changing these and keeping them as 1. so it's easier to update and add/remove steps in the future if needed.

Author: ahh, i had no idea why they were all `1.` but I saw it didn't matter when rendered. TY!

- The code mappings for the project are incorrect.
- The issue was created prior to the integration being set up.

## Where It's Applied

Once suspect commit authors are identified, they're applied in the following ways:

### Suggested Assignees

> On the **Issue Details** page, you'll see suggested assignees based on [ownership rules](/product/issues/ownership-rules/) matching the event, and suspect commits. You can manually assign the issue to a suggested assignee by selecting an option on the dropdown.

Reviewer: ```suggestion
On the **Issue Details** page, you'll see suggested assignees based on [ownership rules](/product/issues/ownership-rules/) matching the event, and suspect commits. You can manually assign the issue to a suggested assignee by selecting one from the dropdown.
```

![Avatar icons of suggested assignees and ability to click on them for assignment.](/product/issues/ownership-rules/suggested_assignees.png)

### Auto-Assignment

You can automatically assign issues to their suspect commit authors by enabling the "Auto-assign to suspect commits" setting in **[Project] > Settings > Ownership Rules**.

![Select from dropdown to automatically assign issues to their suspect commit authors.](/product/issues/ownership-rules/auto-assign-suspect-commits.png)

If an issue has already been assigned manually, new events will not change the current assignee. If an issue has not been assigned and a suspect commit is found, Sentry assigns it to the commit author.

> #### Limitations

Reviewer: ```suggestion
## Limitations
```
Just so this shows up in the right sidenav

> - If auto-assignment is enabled but the Suspect Committer is not in the Sentry organization, we will not be able to auto-assign the issue to them.

Reviewer: ```suggestion
- If auto-assignment is enabled but the suspect commit author is not in the Sentry organization, we will not be able to auto-assign the issue to them.
```
